Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -261,23 +261,67 @@ NS_SWIFT_NAME(Firestore)
 
 #pragma mark - Bundles
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleData Data from the bundle to be loaded.
+ * @return A `FIRLoadBundleTask` (`LoadBundleTask` in Swift) which allows registered observers
+ * to receive progress updates and completion or error events.
+ */
 - (FIRLoadBundleTask *)loadBundle:(NSData *)bundleData NS_SWIFT_NAME(loadBundle(_:));
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleData Data from the bundle to be loaded.
+ * @param completion A block to execute when loading is in a final state. The `error` parameter
+ * will be set if the block is invoked due to an error. If observers are registered to the
+ * `FIRLoadBundleTask` being returned, this block will be called after all observers are notified.
+ * @return A `FIRLoadBundleTask` (`LoadBundleTask` in Swift) which allows registered observers
+ * to receive progress updates and completion or error events.
+ */
 - (FIRLoadBundleTask *)loadBundle:(NSData *)bundleData
                        completion:(nullable void (^)(FIRLoadBundleTaskProgress *_Nullable progress,
                                                      NSError *_Nullable error))completion
     NS_SWIFT_NAME(loadBundle(_:completion:));
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleStream An input stream from which the bundle can be read.
+ * @return A `FIRLoadBundleTask` (`LoadBundleTask` in Swift) which allows registered observers
+ * to receive progress updates and completion or error events.
+ */
 - (FIRLoadBundleTask *)loadBundleStream:(NSInputStream *)bundleStream NS_SWIFT_NAME(loadBundle(_:));
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleStream A input stream from which the bundle can be read.
+ * @param completion A block to execute when the loading is in a final state, the `error` parameter
```

Comment:
"...in a final state. The error block..."
(not a comma)

Reply:
Done.